### PR TITLE
add Docker file

### PR DESCRIPTION
**Pull request by ptolemy428**
Move packer docker file into the cultureamp repo.

As part of this change, I had a look at the possibility of using the alpine linux as the base image for this. But decided not to use it for this PR. 

Some thoughts on alpine base image:

Overall alpine linux base image is a good one to have, as it significantly reduces the size of the image and speed up the docker builds. However, we need a bit more time to come up with a reasonable CA base image. As the default base image is so minimal that requires some extra work for it to be useful for building our tools/app images. I Will work on this as a side task to create a candidate for our (alpine based )base image.

### CODE REVIEW

**Comment by ptolemy428** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;w108xfNK&#x2F;187-update-packer-to-0-10-0"> Update Packer to 0.10.0</a>

**Comment by ptolemy428** [comment]
Have run few baking jobs with Packer 0.10.0. So far, all good.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM buildpack-deps:jessie
```
Comment (hebbo) [inline Dockerfile]: That base Docker image does not sound very official. Can you add a bit if background on why we have chosen this one? Wouldn't a basic Alpine image suffice here?

Comment (ptolemy428) [inline Dockerfile]: The buildpack-deps:jessie is an official image. It is used as the base for various official images such as golang and ruby. It provides a general purpose base image which provides good compatibility with most of the modern apps.
https://hub.docker.com/_/ruby/
https://hub.docker.com/_/golang/

For the reason why I didn't switch to alpine base as part of this PR, please refer to the comments made in the description of this PR. :)

Comment (hebbo) [inline Dockerfile]: Thanks for the clarification Larry.
From the description I understood we would be using the basic Alpine image and create a more suitable one later.

:+1:

Comment (ptolemy428) [inline Dockerfile]: I see where you're coming from now. I originally just wanted to document my thoughts on the alpine base image in the description. I've now updated the wording to be more clear.

**Pull request merged**